package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "a11y-speak": "git+https://github.com/Yoast/a11y-speak.git#master",
     "algoliasearch": "^3.22.3",
+    "grunt-scss-to-json": "^1.0.1",
```

Comment:
This is also in the optionalDepdendencies, I know @atimmer moved it there because of issues on Travis, not sure it should be specified in both the dependencies and the optionalDepdendencies.